src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -1140,8 +1140,6 @@ public void sendMap(@NotNull MapView map)
 	@Override
 	public void updateInventory()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
```

Comment:
Empty methods are not a good practice.
If a method doesn't execute any relevant code then it should have a comment explaining why at _minimum_.

Reply:
ok, I added a small explanation